config-system/pom.xml

```diff
@@ -31,6 +31,10 @@
     </parent>
 
     <dependencies>
+        <dependency>
+            <groupId>com.github.cafapi.util</groupId>
+            <artifactId>util-naming</artifactId>
```

Comment:
This dependency is listed as provided in lots of the other pom files but not here, why is there this inconsistency? If you are going to change this please explain why you are making the change, I do not know which is correct so I want to know why there is a difference.

Reply:
The newly added dependencies are referred from build log.

http://sou-jenkins2.hpeswlab.net/job/CAFapi/job/CAFapi~caf-common~develop~CI/14/consoleFull
Here it is missed to add provided.  Fixed now.
The new dependencies added as per the build log under "Used undeclared dependencies found:".
The scope defined in the log is the correct one.